pkg/proxy/topology.go

```diff
@@ -69,12 +69,38 @@ func FilterTopologyEndpoint(nodeLabels map[string]string, topologyKeys []string,
 		for _, ep := range endpoints {
 			topology := ep.GetTopology()
 			if value, found := topology[key]; found && value == topologyValue {
-				filteredEndpoint = append(filteredEndpoint, ep)
+				filteredEndpoints = append(filteredEndpoints, ep)
 			}
 		}
-		if len(filteredEndpoint) > 0 {
-			return filteredEndpoint
+		if len(filteredEndpoints) > 0 {
+			return filteredEndpoints
 		}
 	}
-	return filteredEndpoint
+	return filteredEndpoints
+}
+
+// FilterLocalEndpoint returns the node local endpoints based on configured
+// InternalTrafficPolicy.
+//
+// If InternalTrafficPolicy is Local, only return the node local endpoints.
+//
+// FilterLocalEndpoint should never be called when InternalTrafficPolicy is All.
```

Comment:
ss/All/Cluster

Reply:
but the code handles this case fine?  Just remove comment?

Reply:
removed comment